src/main/zapHomeFiles/hud/utils.js

```diff
@@ -246,7 +243,20 @@ var utils = (function() {
 
 		return Promise.all(promises)
 	}
-	
+
+	/*
+	 * Backs up the frames tools to ZAP
+	 */
+	function backupFrame(frame) {
```

Comment:
Likewise hear I think it would be more clear if the function was named `backupFrameTools` seeing as thats what were saving.

Reply:
I'd rather not change this, as we might well back up other frame related data such as the frame position in the future.

Reply:
That make sense. Then maybe just in the comment above describing what "backup" means would be good.

Reply:
ah you did that already!